src/app/landing-page/info-card/info-card.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component, Input} from '@angular/core';
+import {NgClass} from "@angular/common";
+
+@Component({
+  selector: 'app-info-card',
+  standalone: true,
+  imports: [
+    NgClass
+  ],
+  templateUrl: './info-card.component.html',
+  styleUrl: './info-card.component.scss'
+})
+export class InfoCardComponent {
+  @Input() ellipseColorClass?: string;
```

Comment:
Zamiast string możesz podać konkretne wartości jakie przyjmuje ten input. Możesz też dodać wartość domyślną, jeśli ma to sens.

```ts
@Input() ellipseColorClass?: 'light-green-ellipse' | 'dark-green-ellipse' = 'light-green-ellipse';
```